flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/FlinkOperatorITCase.java

```diff
@@ -114,6 +114,7 @@ private static FlinkDeployment buildSessionCluster() {
         resource.setCpu(1);
         JobManagerSpec jm = new JobManagerSpec();
         jm.setResource(resource);
+        jm.setReplicas(1);
```

Comment:
Why we have to set the replicas explicitly?

Reply:
Without this setting, `replicas` in `JobManagerSpec` will be zero be default. That's why the IT test failed.
So do you mean we should change the default value of `replicas` in the class? It seems to be a better solution.

Reply:
Yes. I believe the `replicas` is not a field which should be configured explicitly. In most cases, no matter the HA enabled or not, they do not need to set this. Unless they really want a standby JobManager to get faster recovery.

Maybe we could also remove the `replicas` in the examples and `e2e-tests`.

Reply:
Would you like to change the default value of `replicas` in this PR or it will be done in a follow-up ticket?

Reply:
I checked the usage of `replicas` and I think it is better to leave it to another ticket. The change is not big but I hope to make this PR more concentrated on CI workflow improvement.